TESTING.md

```diff
@@ -72,53 +64,6 @@ To run just a single test file in WebdriverIO, update the `specs` config in the
 1. Scroll to the scripts section and click the ▶ button next to `wdio-local`
 1. Select `Debug 'wdio-local'`
 
-### Protractor
-
-#### Visual Studio Code
-
-##### Setting up Vscode for e2e debugging.
```

Comment:
Sorry for my Protractor/WDIO ignorance, but all this paragraph is not relevant anymore? Isn't there any setup for debugging WDIO?

Reply:
Using Webstorm you can debug really easily, and you don't need to add any specific configuration to do it.

@ngaruko was working on a PR to be able to debug wdio e2e tests, he had some problems with it, but a new PR will be created. Once that is done we can add a new section to the doc 🙂

Reply:
Excellent, thank you for the clarification! Also, it's worth mentioning that many colleagues still use VS Code, in case they need to do anything specific to have the debugging working correctly.

Reply:
You are right. But I can't help here because I never learned how to debug using VS code 😅